src/InventoryManager.cpp

```diff
@@ -130,7 +130,21 @@ int InventoryManager::userInput() {
                                  name.c_str(), value.c_str());
             }
             break;
+        case 'c':
+        case 'C':
+            std::cin.clear();
+            std::cin.ignore(10000, '\n');
 
+            std::cout << "Account name: ";
+            std::cin >> name;
+            std::cout << "New account tpye: ";
+            std::cin >> category;
+
+            if (updatePermission(name, category)) {
```

Comment:
Very minor suggestion:

For the other cases that have permission checks, such as add and remove, we output a warning message.

I think it'd make sense to do something like:
``` 
if (updatePermission(name, category)) {
    ...
} else {
    Logger::logWarn("User %s does not have the required permissions to change permissions of user '%s'.", current_user->name.c_str(), name.c_str());
}
```

I don't think this is that important, but I feel like it would be consistent with our other cases.  Feel free to mark as resolved.